static/index.html

```diff
@@ -199,13 +202,13 @@ <h3>Olimex ESP32 Power-over-Ethernet ISO</h3>
         <ul>
           <li>
             <a
-              href="https://www.olimex.com/Products/IoT/ESP32/ESP32-POE-ISO/open-source-hardware"
+              href="https://www.olimex.com/Products/IoT/ESP32/ESP32-POE-ISO-EA/open-source-hardware"
               >Olimex</a
             >
           </li>
           <li>
             <a
-              href="https://www.mouser.com/ProductDetail/Olimex-Ltd/ESP32-POE-ISO"
+              href="https://www.mouser.com/ProductDetail/Olimex-Ltd/ESP32-POE-ISO-EA?qs=%252B6g0mu59x7KfK%2FIl1ERS8Q%3D%3D"
```

Comment:
Do we need the qs part?

Reply:
Yes, otherwise its a 404

Reply:
Noticed this:

![CleanShot 2022-09-06 at 21 04 45](https://user-images.githubusercontent.com/1444314/188766307-f9a3ea17-7632-4435-8562-90f2990c525d.png)